shopfloor/tests/test_actions_data_detail.py

```diff
@@ -202,6 +203,18 @@ def test_data_move_line_package(self):
             "progress": 30.0,
         }
         self.assertDictEqual(data, expected)
+        data = self.data_detail.move_line(move_line, with_package_move_line_count=True)
+        package_src = expected["package_src"]
+        package_src.update({"move_line_count": 1})
+        package_dest = expected["package_dest"]
+        package_dest.update({"move_line_count": 1})
+        expected.update(
```

Comment:
The tests in this file have all the same workflow: generate some data, check it matches a schema and validate with some static content.
Here the data is generated and then modified, which does not make sense to me.

Reply:
the call self.data_detail.move_line(move_line, with_package_move_line_count=True)
creates the same data it only adds "move_line_count" on package_src and package_dest.
That's why i don't need to define the expected data again.

Reply:
See https://github.com/OCA/wms/pull/719#discussion_r1296758759